docs/api/meetings.md

```diff
@@ -0,0 +1,636 @@
+| `{SOURCE_TYPE}` | The type of source you want to filter for. This value is **only** used by `virtual` sources. Possible values include `zoom`, `slack`, `teams`, and `skype`. |
+| `{TEAM_ID}` | The ID of the team that you want to filter for. |
+| `{TEAM_NAME}` | The name of the team that you want to filter for. | 
+| `{LIMIT}` | Specifies the number of meetings returned. | 
+| `{PAGE}` | Specifies the offset of the pages of meetings returned. |
+
+### Request
+
+The following request will retrieve the last two meetings for the Agenda Documentation team.
+
+```shell
+curl -X GET https://api.agenda.com/meetings?limit=2&team-id=7 \
+-H 'Authorization: Bearer {ACCESS_TOKEN}' \
+-H 'Client-ID: {API_KEY}'
+```
+
+### Response
+
+A successful response returns HTTP status 200 with a list of meetings based on the query parameters provided in the request path.
+
+```json
+{
+    "records": [
+        {
+            "id": 369,
+            "source": {
+                "name": "Solis",
+                "category": "in-person",
+                "id": 8
+            },
+            "invitees": [
+                {
+                    "name": "Stephen Chan",
+                    "email": "stephenc@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 13
+                },
+                {
+                    "name": "Pierre Noble",
+                    "email": "pierren@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 35
+                },
+                {
+                    "name":"Kyouko Sakura",
+                    "email": "kyoukos@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 40
+                },               
+                {
+                    "name":"Dmitry Podkolzin",
+                    "email": "dmitryp@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 79
+                },
+                {
+                    "name":"Jessica Smith",
+                    "email": "jessicas@agenda.com",
+                    "external": false,
+                    "status": "accepted",
+                    "team": 7,
+                    "id": 133
+                }
+            ],
+            "name": "Standup",
+            "description": "Daily standup for the Agenda Documentation team. Use this meeting to discuss wins, challenges, and other documentation-related topics.",
+            "createdAt": 1641289072,
+            "updatedAt": 1641289072,
+            "createdBy": 35,
+            "updatedBy": 35,
+            "meetingTime": 1667556000,
```

Comment:
In addition to `meetingTime`, we should also have something like `duration` to indicate how long the meeting is scheduled to last for. Alternatively we could replace `meetingTime` with `meetingStart`, and add a second timestamp `meetingEnd` if that's seems more realistic.

Reply:
I think duration makes more sense - when you normally plan a meeting, you usually say a meeting that starts at 3PM that lasts 45 minutes, not a meeting that starts at 3PM and ends at 3:45PM. Let's add that field in as well.